frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -12,7 +12,7 @@ import NamespacedPage from '../NamespacedPage';
 import ProjectsExistWrapper from '../ProjectsExistWrapper';
 import DefaultPage from '../DefaultPage';
 import { getCheURL } from './topology-utils';
-import TopologyDataController, { RenderProps } from './TopologyDataController';
+import ConnectedTopologyDataController, { RenderProps } from './TopologyDataController';
```

Comment:
I'm getting eslint error `Using exported name 'TopologyDataController' as identifier for default` because I'm also exporting the TopologyDataController from that file.

Reply:
Weird. Let me look into that.